### ISSUE TITLE
Teamskeet.com (NSFW) (unbreak and tracker block)

### ISSUE THREAD

**Issue by bobsage123** [uBlockOrigin/uAssets#3251]
MVPSHosts breaks all video thumbnails on the home page by blocking 
`||cdn.teamskeetimages.com^`

Also found the below tracking URL running in the background unblocked. I tested blocking it and it did not break site functionality. Would suggest we put this in the main ublock filter (unless the ublock main filter purposely does not include 1st party scripts\XHRs?)
`https://tracking.teamskeet.com/event.json`

**Comment by okiehsch** [comment]
`uBO-unbreak` is only for site breakage that is caused by the default setup.
`MVPSHosts` is not part of the default setup, so no filter will be added.
Add
`@@||cdn.teamskeetimages.com^$image,domain=teamskeet.com`
to your filter list to fix the issue.

**Comment by bobsage123** [comment]
So it looks like maybe this is affecting site functionality.
All the video images don't appear on this if the ublock privacy feature is on which has:

`teamskeet.com##+js(abort-on-property-read.js, $.tstracker)`

https://www.teamskeet.com/t1/updates/?site=tp&view=rating#1
